pkg/capi/util/validators.ts

```diff
@@ -0,0 +1,121 @@
+import { Validator, ValidationOptions } from '@shell/utils/validators/formRules';
+import { Translation } from '@shell/types/t';
+
+// const stringFormats = {
```

Comment:
is this intentional?

Reply:
No, sorry - I updated the PR to include the validation I had started writing out here & removed this.